src/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ if(CMAKE_EXTRA_GENERATOR OR MSVC_IDE)
     list(APPEND CUKE_SOURCES ${CUKE_HEADERS})
 endif()
 
+string(REGEX REPLACE "([^;]+)" "${CMAKE_CURRENT_SOURCE_DIR}/\\1" COVER_SOURCES "${CUKE_SOURCES}")
+file(WRITE sourceFileList "${COVER_SOURCES}" )
```

Comment:
It was just a thing I was playing with and even deleted from my repo - just forgot pushing deletion commit here